Makefile

```diff
@@ -98,28 +135,44 @@ govet:
 
 .PHONY: golint
 golint:
-	@go get github.com/golang/lint/golint
 	$(eval LINT_LOG := $(shell mktemp -t golint.XXXXX))
 	@cat /dev/null > $(LINT_LOG)
 	@$(foreach pkg, $(PACKAGES), golint $(pkg) | $(FILTER_LINT) >> $(LINT_LOG) || true;)
 	@[ ! -s "$(LINT_LOG)" ] || (echo "golint failed:" | cat - $(LINT_LOG) && false)
 
 .PHONY: staticcheck
 staticcheck:
-	@go get honnef.co/go/tools/cmd/staticcheck
 	$(eval STATICCHECK_LOG := $(shell mktemp -t staticcheck.XXXXX))
 	@staticcheck $(PACKAGES) 2>&1 | $(FILTER_LINT) > $(STATICCHECK_LOG) || true
 	@[ ! -s "$(STATICCHECK_LOG)" ] || (echo "staticcheck failed:" | cat - $(STATICCHECK_LOG) && false)
 
 .PHONY: errcheck
 errcheck:
-	@go get github.com/kisielk/errcheck
 	$(eval ERRCHECK_LOG := $(shell mktemp -t errcheck.XXXXX))
 	@errcheck $(ERRCHECK_FLAGS) $(PACKAGES) 2>&1 | $(FILTER_LINT) | $(FILTER_ERRCHECK) > $(ERRCHECK_LOG) || true
 	@[ ! -s "$(ERRCHECK_LOG)" ] || (echo "errcheck failed:" | cat - $(ERRCHECK_LOG) && false)
 
+.PHONY: verify_version
+verify_version:
+	@if [ "$(INTHECODE_VERSION)" = "$(CHANGELOG_VERSION)" ]; then \
+		echo "yarpc-go: $(CHANGELOG_VERSION)"; \
+	elif [ "$(INTHECODE_VERSION)" = "$(CHANGELOG_VERSION)-dev" ]; then \
+		echo "yarpc-go (development): $(INTHECODE_VERSION)"; \
+	else \
+		echo "Version number in version.go does not match CHANGELOG.md"; \
+		echo "version.go: $(INTHECODE_VERSION)"; \
+		echo "CHANGELOG : $(CHANGELOG_VERSION)"; \
+		exit 1; \
+	fi
+
+.PHONY: lintbins
+lintbins:
+	@go get github.com/golang/lint/golint
+	@go get honnef.co/go/tools/cmd/staticcheck
+	@go get github.com/kisielk/errcheck
```

Comment:
If you put all the packages in a single `go get` invocation, it's a bit faster (though possibly not for these specific deps) - it lets the go tool compute the whole dependency graph and go get everything just once.